### PR TITLE
Added validation for Import by IP and new task in orchestration

### PR DESCRIPTION
**Pull request by arabind**
@revinjacob : Please review and merge the code which contains:
1- Validation for Import by IP and new task in orchestration.

### CODE REVIEW

File: client/cat3/main.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
 		<script src="lib/angular-sanitize/angular-sanitize.min.js"></script>
 		<script src="customlib/angular-messages.js"></script>
 		<script src="lib/lodash/lodash/lodash.min.js"></script>
+		<script src="lib/ng-file-upload/ng-file-upload.js"></script>
 		<!-- Application configuration -->
```
Comment (revinjacob) [inline client/cat3/main.html]: minified js file from lib should be included

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTask.html [hunk 2]
```diff
@@ -1,325 +1,344 @@
...
-			<i class="fa fa-plus"></i>&nbsp;
-			Create {{taskType}} job
-		</h4>
-		<h4 class="modal-title" ng-if="isEditMode">
-			<i class="fa fa-edit"></i>&nbsp;
-			Edit {{taskType}} job - {{name}}
-		</h4>
-	</div>
+	<form name="newtaskForm" novalidate>
+		<div class="modal-header">
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTask.html]: form name should follow lower camelcase naming standards

**Comment by revinjacob** [comment]
pls resolve conflicts

**Comment by arabind** [comment]
@revinjacob : Please merge.
I have handled comments for PR#359

**Pull request merged**